colanderalchemy/schema.py

```diff
@@ -130,14 +148,17 @@ class MyModel(Base):
 
         # Obtain configuration specific from the mapped class
         kwargs.update(getattr(self.inspector.class_, self.ca_class_key, {}))
+        includes = includes or kwargs.pop('includes', {})
+        excludes = excludes or kwargs.pop('excludes', {})
+        overrides = overrides or kwargs.pop('overrides', {})
```

Comment:
I was keeping it in with the existing ``unknown =`` line; but yes, it could have been removed. 

However, on balance, the logic here needs to revisited.  If ``includes`` gets provided as an argument to this function call and it's also present in ``kwargs``(eg coming in from ``__colanderalchemy_config__``) then it'll be left in ``kwargs`` and thus passed through to ``SchemaNode.__init__`` .  This will technically have no effect given Colander's current code but it's unnecessary processing & might cause side effects later.  I'm updating the PR now.